src/main/webapp/WEB-INF/pages/priceHistory.jsp

```diff
@@ -0,0 +1,31 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="product" type="com.es.phoneshop.model.product.Product" scope="request"/>
+<tags:master pageTitle="Price History">
+    <br>
+    <h1>${product.description}</h1>
+    <table>
+        <thead>
+        <tr>
+            <td>Date</td>
+
+            <td>Price</td>
+        </tr>
+        </thead>
+        <c:forEach var="priceHistoryItem" items="${product.priceHistory}">
+            <tr>
```

Comment:
If you use thead, then please add tbody as well.

Reply:
done (delete thread)